### PR TITLE
Fix: Missing deployments for LiquidityBootstrappingPoolFactory

### LINKED ISSUE #213: Fix: Missing deployments for LiquidityBootstrappingPoolFactory
## To Reproduce
- try to deploy the LBP contracts to rinkeby using the current deploy script
- the same applies to deployments to mainnet

## Expected behavior
- it is deployed

## Actual behavior
- it cannot be deployed since the `LiquidityBootstrappingPoolFactory` contract cannot be retrieved

### LINKED ISSUE #217: LBPManager should have a getSwapEnabled function
LBPManager has a function `setSwapEnabled` but no `getSwapEnabled`.  This is awkward for users of the contract who want to work with the swapEnabled state.

### PR DESCRIPTION
**Pull request by Zitzak**
## What did I do?
- added `@balancer-labs/v2-deployments` package
- updated `03_deploy_lbp_contracts` so it dynamically get the network name and `liquidityBootstrappingPoolFactory` address
- deployed new contract with changes from the audit
- verified the deployed contracts
- added ABIs

## Implements issue:
- #213 

closes #213

### CODE REVIEW

**Comment by Zitzak** [comment]
Screen Shot 2021-11-03 at 21 48 14

**Review by BritikovKI — APPROVED** [review]

File: deploy/03_deploy_lbp_contracts.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
 const deployFunction = async ({ getNamedAccounts, deployments, ethers }) => {
   const { deploy } = deployments;
   const { root } = await getNamedAccounts();
   const safeInstance = await ethers.getContract("Safe");
-  const lbpFactoryInstance = await ethers.getContract(
-    "LiquidityBootstrappingPoolFactory"
+  const liquidityBootstrappingPoolFactoryTaskId =
+    "20210721-liquidity-bootstrapping-pool";
+  const contractName = "LiquidityBootstrappingPoolFactory";
+  const networkName = hre.network.name;
```
Comment (fabianschu) [inline deploy/03_deploy_lbp_contracts.js:10]: funky stuff 💪🏼

Comment (fabianschu) [reply deploy/03_deploy_lbp_contracts.js:10]: btw you can get the `network`object the same way you get the `ethers` and `deployments` objects in line 3

**Review by fabianschu — APPROVED** [review]

**Review by dkent600 — APPROVED** [review]

File: deploy/03_deploy_lbp_contracts.js (hunk 1 above)
Comment (Zitzak) [reply deploy/03_deploy_lbp_contracts.js:10]: Yeah just came accros the deployment package from Balancer and it worked great :D

**Pull request merged**